README.md

```diff
@@ -55,3 +55,13 @@ docker build -t jupyter/demo .
 docker build -t jupyter/minimal common/
 ```
 
+#### `jupyter/minimal`
+You will need to run this code snippet before building the docker image
+```
+git clone https://github.com/ericdill/scikit-xray-examples
```

Comment:
Good point. Let's do one better. If the directory exists, git remote update. If the directory does not exist, git clone. Every update will require a new build, and I think that our main reason for rebuilding will be an update to the git repo(s).